website/views/options/settings.jade

```diff
@@ -379,12 +379,12 @@ script(id='partials/options.settings.subscription.html',type='text/ng-template')
 
           .form-inline
             .form-group
-              input.form-control(type='text', ng-model='_subscription.coupon', placeholder='Promo Code')
+              input.form-control(type='text', ng-model='_subscription.coupon', placeholder= env.t('couponPlaceholder'))
```

Comment:
I think that's reasonable, I'm just not sure if there's a difference between the two. @lemoness can make the final decision. I'd be fine leaving it at "Enter Coupon Code".

The other place the coupon code text is used is on the [Settings > Coupon page](https://habitrpg.com/#/options/settings/coupon)

![screen shot 2015-05-03 at 2 40 00 pm](https://cloud.githubusercontent.com/assets/2916945/7446575/588ed302-f1a2-11e4-91fb-5b667cf6dc6b.png)